kotlin-source/build.gradle

```diff
@@ -32,15 +32,17 @@ dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.11'
 
```

Comment:
Should probably add this too:
```
testCompile group: 'org.jetbrains.kotlin', name: 'kotlin-test', version: kotlin_version
```
because Kotlin's test framework has been removed from Corda.